pandas/core/frame.py

```diff
@@ -2323,6 +2323,7 @@ def _from_arrays(
         index,
         dtype: Dtype | None = None,
         verify_integrity: bool = True,
+        consolidate: bool = True,
```

Comment:
Is this needed now since it's always `True`?

Reply:
nope, removed